src/game.js

```diff
@@ -421,12 +419,12 @@ var Game = Class.create({
 
 		this.nextCreature();
 
-		G.resizeCombatFrame(); // Resizes while the game is starting
+		G.resizeCombatFrame(); // Resize while the game is starting
 		G.UI.resizeDash();
 
-		// Resizes event
+		// Handle resize events
```

Comment:
The other guy recently added a bunch of `s` to each verb, we need to figure out the standard way of doing things regarding this matter. I'm guessing no `s` is the typical way to go, it kinda felt weird that way tbh :D